x-pack/qa/sql/src/main/resources/ip.csv-spec

```diff
@@ -0,0 +1,149 @@
+//
+// Tests for IP fields
+//
+
+selectAll
+SELECT * FROM logs ORDER BY id LIMIT 10;
+
+       @timestamp       |   bytes_in    |   bytes_out   |   client_ip   |  client_port  |    dest_ip    |      id       |    status     
+------------------------+---------------+---------------+---------------+---------------+---------------+---------------+---------------
+2017-11-10T21:15:54.130Z|47             |388            |10.0.1.166     |9152           |172.27.1.1     |1              |OK             
+2017-11-10T21:15:39.988Z|29             |374            |10.0.1.166     |31693          |172.27.1.1     |2              |OK             
+2017-11-10T21:15:39.611Z|35             |303            |10.0.1.166     |23625          |172.27.1.1     |3              |OK             
+2017-11-10T21:15:39.601Z|36             |312            |10.0.1.166     |9932           |172.27.1.1     |4              |OK             
+2017-11-10T21:15:40.792Z|35             |344            |10.0.1.166     |22695          |172.27.1.1     |5              |OK             
+2017-11-10T21:15:40.805Z|31             |503            |10.0.1.166     |59811          |172.27.1.1     |6              |OK             
+2017-11-10T21:15:40.790Z|35             |458            |10.0.1.166     |57372          |172.27.1.1     |7              |OK             
+2017-11-10T21:15:41.799Z|35             |281            |10.0.1.166     |17370          |172.27.1.1     |8              |OK             
+2017-11-10T21:15:41.867Z|46             |231            |10.0.1.166     |65004          |172.27.1.1     |9              |OK             
+2017-11-10T20:36:07.364Z|40             |506            |10.0.1.166     |22661          |172.27.1.1     |10             |OK  
+;
+
+selectIpField
+SELECT client_ip, dest_ip FROM logs ORDER BY id LIMIT 10;
+
+   client_ip   |    dest_ip    
+---------------+---------------
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1     
+10.0.1.166     |172.27.1.1 
+;
+
+orderByIpField
+SELECT client_ip FROM logs ORDER BY client_ip LIMIT 10;
+
+   client_ip   
+---------------
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122     
+10.0.1.122 
+;
+
```

Comment:
Maybe also add an ORDER BY dest_ip so that we can see the IPv6 ordering.

Reply:
Done.